ladybug_geometry/geometry2d/pointvector.py

```diff
@@ -404,3 +404,9 @@ def __sub__(self, other):
     def __repr__(self):
         """Point2D representation."""
         return 'Point2D (%.2f, %.2f)' % (self.x, self.y)
+
+    def __lt__(self, other):
+        """Used for storing points in external 1D data structure libraries
+        """
+        if isinstance(other, Vector2D):
+            return self.x < other.x
```

Comment:
Can we add a `__gt__` method that returns the inverse of this.  Also, can we add the same types of methods to the `Point3D` class? I know that you wanted to keep the impact of this method minimal since it doesn't follow a particular historical precedent but the inconsistency that exists here now will present bigger issues than the break with historical precedent.